client/src/views/Report/GroupedBarChart.js

```diff
@@ -17,6 +17,13 @@ export default function GroupedBarChart(props) {
         data,
     } = props;
 
+
+    if (data.values.length <= 0) {
```

Comment:
Are you sure data.values is defined?

Reply:
The `getSoiTrendData` has ```
return ({
            values,
            columns: [
                'Total Closed',
                'Closed On',
            ],
            colors: {
                'Total Closed': '#41cf76',
                'Closed On': '#ef8c00',
            },
        })
```

and data is the returned value of above calculation. So, I think data.values will be defined.

Do, I need to check if data.values exists as well ?